### PR TITLE
initial react files

### PR DESCRIPTION
**Pull request by mcraepv**

### CODE REVIEW

File: frontend/src/setupTests.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```
Comment (mmevans) [inline frontend/src/setupTests.js]: could you remove these comments?

File: frontend/src/logo.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...21.2 430c-9.3-9.6-18.6-20.3-27.8-32 9 .4 18.2.7 27.5.7 9.4 0 18.7-.2 27.8-.7-9 11.7-18.3 22.4-27.5 32zm-74.4-58.9c-14.2-2.1-27.9-4.7-41-7.9 3.7-12.9 8.3-26.2 13.5-39.5 4.1 8 8.4 16 13.1 24 4.7 8 9.5 15.8 14.4 23.4zM420.7 163c9.3 9.6 18.6 20.3 27.8 32-9-.4-18.2-.7-27.5-.7-9.4 0-18.7.2-27.8.7 9-11.7 18.3-22.4 27.5-32zm-74 58.9c-4.9 7.7-9.8 15.6-14.4 23.7-4.6 8-8.9 16-13 24-5.4-13.4-10-26.8-13.8-39.8 13.1-3.1 26.9-5.8 41.2-7.9zm-90.5 125.2c-35.4-15.1-58.3-34.9-58.3-50.6 0-15.7 22.9-35.6 58.3-50.6 8.6-3.7 18-7 27.7-10.1 5.7 19.6 13.2 40 22.5 60.9-9.2 20.8-16.6 41.1-22.2 60.6-9.9-3.1-19.3-6.5-28-10.2zM310 490c-13.6-7.8-19.5-37.5-14.9-75.7 1.1-9.4 2.9-19.3 5.1-29.4 19.6 4.8 41 8.5 63.5 10.9 13.5 18.5 27.5 35.3 41.6 50-32.6 30.3-63.2 46.9-84 46.9-4.5-.1-8.3-1-11.3-2.7zm237.2-76.2c4.7 38.2-1.1 67.9-14.6 75.8-3 1.8-6.9 2.6-11.5 2.6-20.7 0-51.4-16.5-84-46.6 14-14.7 28-31.4 41.3-49.9 22.6-2.4 44-6.1 63.6-11 2.3 10.1 4.1 19.8 5.2 29.1zm38.5-66.7c-8.6 3.7-18 7-27.7 10.1-5.7-19.6-13.2-40-22.5-60.9 9.2-20.8 16.6-41.1 22.2-60.6 9.9 3.1 19.3 6.5 28.1 10.2 35.4 15.1 58.3 34.9 58.3 50.6-.1 15.7-23 35.6-58.4 50.6zM320.8 78.4z"/><circle cx="420.9" cy="296.5" r="45.7"/><path d="M520.5 78.1z"/></g></svg>
```
Comment (mmevans) [inline frontend/src/logo.svg]: any logo that is unused can be removed! I think there are a couple like this one that don't need to be checked in

File: frontend/src/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want to start measuring performance in your app, pass a function
```
Comment (mmevans) [inline frontend/src/index.js]: auto generated comments like this one are good to remove

File: frontend/src/index.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
```
Comment (mmevans) [inline frontend/src/index.css]: you can get rid of this file as well!

File: frontend/src/App.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
```
Comment (mmevans) [inline frontend/src/App.js:2]: all of the text, images, logos, etc can be removed. I usually either no text or a single placeholder that says <div>React o-shop</div>

File: frontend/src/App.css [hunk 6]
```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
```
Comment (mmevans) [inline frontend/src/App.css]: this file can be removed

File: frontend/src/setupTests.js (hunk 1 above)
Comment (mcraepv) [reply frontend/src/setupTests.js]: Sure thing

File: frontend/src/logo.svg (hunk 2 above)
Comment (mcraepv) [reply frontend/src/logo.svg]: Removed the svg and the other react logos in the public directory

File: frontend/src/index.js (hunk 3 above)
Comment (mcraepv) [reply frontend/src/index.js]: Combed files for auto-comments and deleted

File: frontend/src/index.css (hunk 4 above)
Comment (mcraepv) [reply frontend/src/index.css]: Deleted

File: frontend/src/App.css (hunk 6 above)
Comment (jasonephraim) [reply frontend/src/App.css:2]: removed

File: frontend/src/App.js (hunk 5 above)
Comment (mcraepv) [reply frontend/src/App.js:2]: Deleted those logos and imports throughout

Comment (jasonephraim) [reply frontend/src/App.js:5]: boilerplate removed and replaced

File: frontend/src/App.css (hunk 6 above)
Comment (mcraepv) [reply frontend/src/App.css]: deleted this file and imports

Comment (jasonephraim) [reply frontend/src/App.css]: @mcraepv Sorry, where would I find the imports?

File: frontend/public/manifest.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    },
+    {
+      "src": "logo192.png",
```
Comment (mmevans) [inline frontend/public/manifest.json]: can you empty this array of icons and delete the ones you aren't going to use?

**Review by mmevans — APPROVED** [review]

File: frontend/public/manifest.json (hunk 7 above)
Comment (mcraepv) [reply frontend/public/manifest.json]: Array deleted

**Pull request merged**